### ISSUE TITLE
[WinError 6] The handle is invalid

### ISSUE THREAD

**Issue by smaragdus** [jeanslack/Videomass#34]
I updated [Videomass](https://github.com/jeanslack/Videomass) from version [3.4.2](https://github.com/jeanslack/Videomass/releases/tag/v3.4.2) to version [3.4.5](https://github.com/jeanslack/Videomass/releases/tag/v.3.4.5) and I discovered **YouTube Downloader** module no longer works. Any time I try to download a video I get the following error:

```
URL 1/1
Source: https://www.youtube.com/watch?v=_gjddiSszDw
Destination:  "C:\Users\Yo\Desktop"

[WinError 6] The handle is invalid

[Videomass]: Sorry, task failed !
```

**screen:**

![Videomass 3 4 5 - 2021-09-11 - error - 001](https://user-images.githubusercontent.com/9025757/132932022-6f85b8bb-f167-422f-a09e-0cc6d4830f4a.png)

Also, when I try to use this command:

`View -> Youtube-dl -> Version in Use`

I get the following error:

`[WinError 6] The handle is invalid`

**screen:**

![Videomass 3 4 5 - 2021-09-11 - error - 002](https://user-images.githubusercontent.com/9025757/132932092-7064aebe-6b2f-4cb7-9e45-6d72c8daba09.png)

When I use version [3.4.2](https://github.com/jeanslack/Videomass/releases/tag/v3.4.2) the downloads complete just fine.

When I go to:

`Settings -> Preferences -> youtube-dl`

and try to open the embedded link:

- [Required: Microsoft Visual C++ 2010 Redistributable Package (x86)](https://www.microsoft.com/en-US/download/details.aspx?id=5555)

**screen:**

![Videomass 3 4 5 - 2021-09-11 - error - 004](https://user-images.githubusercontent.com/9025757/132932622-5c12f537-c3ec-44be-bbbd-0e865ad9ce67.png)

it redirects to:

- [Error 404 - Not Found](https://www.microsoft.com/en-us/download/404Error.aspx)

I am using [Videomass](https://github.com/jeanslack/Videomass) on **Windows 8 x64** and I have installed **Microsoft Visual C++ Redistributable Package 2010** version **10.0.40219.325** both **x86** and **x64** releases.

So for the time being I have to downgrade to [Videomass](https://github.com/jeanslack/Videomass) version [3.4.2](https://github.com/jeanslack/Videomass/releases/tag/v3.4.2) which works for me.

**Comment by jeanslack** [comment]
Hi @smaragdus,
I have checked in detail the problems you reported, I will solve them soon with the next release.

**Comment by smaragdus** [comment]
@jeanslack 

Thanks for responding, confirming and fixing the problem.

When there is a new release I will test it

Regards

**Comment by smaragdus** [comment]
@jeanslack 

The problems described above is really solved in version [3.4.6](https://github.com/jeanslack/Videomass/releases/tag/v.3.4.6), screens:

![Videomass 3 4 6 - 2021-09-17 - 001](https://user-images.githubusercontent.com/9025757/133685695-36fef52d-b5c5-46ff-ba38-07bf4c3e337e.png)

![Videomass 3 4 6 - 2021-09-17 - 002](https://user-images.githubusercontent.com/9025757/133686137-3665e088-22cd-4b38-9f74-224e9503e291.png)

The broken link - [Required: Microsoft Visual C++ 2010 Service Pack 1 Redistributable Package (x86)](https://download.microsoft.com/download/1/6/5/165255E7-1014-4D0A-B094-B6A430A6BFFC/vcredist_x86.exe), is also fixed

Thank you for all your work to maintain and improve this project!

Regards

**Comment by jeanslack** [comment]
@smaragdus 

Thank you for using Videomass and reporting this issue.
At your disposal!
